### ISSUE TITLE
[Accessibility] [Linux] Crash when espeak not installed

### ISSUE THREAD

**Issue by RobLoach** [libretro/RetroArch#15328]
When `espeak` is not installed, enabling Accessibility, and then switching menu items will crash RetroArch.

### Steps to Reproduce
1. Remove espeak from your system
1. Run Linux/Unix
2. Enable Accessibility
3. Switch menu items
4. See that it'll crash

### Proposed Fix
1. Make it gracefully fail when espeak isn't available

**Comment by gouchi** [comment]
It seems I can't reproduce the issue but I got this message when I quit RA and Accessibility is enabled but not available.

```
X connection to :0 broken (explicit kill or server shutdown).
X connection to :0 broken (explicit kill or server shutdown).
```

We should investigate [accessibility_speak_unix](https://github.com/libretro/RetroArch/blob/master/frontend/drivers/platform_unix.c#L2795-L2865) function.

**Comment by geekley** [comment]
I believe this is fixed according to this? https://www.libretro.com/index.php/retroarch-1-17-0-release/
> ACCESSIBILITY/TTS: fix target language and missing espeak handling on Linux

**Comment by RobLoach** [comment]
Thanks for the follow up! Confirmed this isn't an issue anymore. It doesn't crash anymore, but just warns that espeak isn't available....

```
[INFO] Spoke: Accessibility
[WARN] Could not execute espeak.
```
